decentraland/unity-explorer
# fix: Spawn points do not work in SDK7

## What does this PR change?
Fix #619 

## How to test the changes?
1. Launch the explorer.
2. Check that the initial position of your avatar is correct.
3. Do different teleports.
4. Check that the new position of your avatar is correct.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
